tool/net/lfuncs.c

```diff
@@ -854,6 +858,22 @@ int LuaUuidV4(lua_State *L) {
   return 1;
 }
 
+int LuaUuidV7(lua_State *L) {
+  char uuid_str[37] = {0};
+  struct timespec now = timespec_real();
+  uint64_t time_now = timespec_tonanos(now);
```

Comment:
This function isn't really intended to be called on an absolute timestamp. Could you use the `tv_sec` and `tv_nsec` fields instead?